platform/macos/godot_application_delegate.h

```diff
@@ -35,18 +35,12 @@
 #import <AppKit/AppKit.h>
 #import <Foundation/Foundation.h>
 
-@interface GodotApplicationDelegate : NSObject <NSUserInterfaceItemSearching, NSApplicationDelegate> {
-	bool high_contrast;
-	bool reduce_motion;
-	bool reduce_transparency;
-	bool voice_over;
-}
+class OS_MacOS;
+
+@interface GodotApplicationDelegate : NSObject <NSUserInterfaceItemSearching, NSApplicationDelegate>
+
+- (GodotApplicationDelegate *)initWithOS:(OS_MacOS *)os;
 
-- (void)forceUnbundledWindowActivationHackStep1;
-- (void)forceUnbundledWindowActivationHackStep2;
-- (void)forceUnbundledWindowActivationHackStep3;
```

Comment:
Moved these to the `GodotApplication` class, as I was exploring removing `NSApplication` loop completely from embedded child process, as it may not be needed.